cli/copy.go

```diff
@@ -94,7 +95,14 @@ func Copy(c *cli.Context) (err error) {
 		if err != nil {
 			return cli.NewExitError(err, 1)
 		}
-		vfile, err = virtualImage.OpenFile(privateKey, c.Args().Get(1))
+
+		// if destination file name is not supplied, use source file name MEN-5463
```

Comment:
The comment is nice to have (thanks!), but the reference to the bug tracker is not needed
```suggestion
		// if destination file name is not supplied, use source file name
```